list-package-updates

```diff
@@ -34,9 +34,8 @@ FILTER=pre_wrap
 SERIES=
 while [[ $1 = -* ]]; do
 case $1 in
-  --bare        ) FILTER=cat;      shift;;
-  -u|--upcoming ) SERIES=upcoming; shift;;
-             -* ) echo "Unsupported option: $1" >&2; usage ;;
```

Comment:
What is the reason for removing the `-u` option here?

More importantly, why is the unsupported option check removed?  Without it the script will hang in an infinite loop if you mistype `--upcomnig` :-p

Also, I'm confused how the commit message "Use a standard SVN layout" explains the changes in this commit...